### PR TITLE
Add product-payload class

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: Payload/ProductPayload.php [hunk 1]
```diff
@@ -38,6 +39,48 @@ public function getCode(): string
...
+
+        return $translations;
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getImages(): array
+    {
+        throw new FeatureNotImplementedException('Images are not implemented');
```
Comment (niklasnatter) [inline Payload/ProductPayload.php:94]: why? 😢

Comment (wachterjohannes) [reply Payload/ProductPayload.php:94]: will come in future ;)

File: Payload/ProductPayload.php [hunk 2]
```diff
@@ -38,6 +39,61 @@ public function getCode(): string
...
+            $variants[] = new ProductVariantPayload($variantPayload['code'], $variantPayload);
+        }
+
+        return $variants;
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getImages(): array
```
Comment (niklasnatter) [inline Payload/ProductPayload.php:92]: would add a test for the sake of this for completion 🙂

File: Payload/ProductTranslationPayload.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SyliusConsumerBundle\Payload;
+
+use Sulu\Bundle\SyliusConsumerBundle\Common\Payload;
+use Sulu\Component\Localization\Localization;
+
+class ProductTranslationPayload
```
Comment (niklasnatter) [inline Payload/ProductTranslationPayload.php:19]: not tested

File: Payload/TaxonPayload.php [hunk 4]
```diff
@@ -78,6 +79,14 @@ public function getTranslations(): array
         return $translations;
     }
 
+    /**
+     * @return mixed[]
+     */
+    public function getImages(): array
```
Comment (niklasnatter) [inline Payload/TaxonPayload.php:85]: not tested

File: Payload/ProductPayload.php (hunk 2 above)
Comment (wachterjohannes) [reply Payload/ProductPayload.php:92]: ?? a test which tests the exception ?? i am not sure if that makes sense 🙈

File: Payload/TaxonPayload.php (hunk 4 above)
Comment (wachterjohannes) [reply Payload/TaxonPayload.php:85]: ?? a test which tests the exception ?? i am not sure if that makes sense 🙈

File: Payload/ProductPayload.php (hunk 2 above)
Comment (niklasnatter) [reply Payload/ProductPayload.php:92]: it would make sense because it forces you to think about the testcase when implementing the method 🙂

**Pull request merged**